### PR TITLE
Fix warning about initialContent and intialprops in Filter

### PR DESCRIPTION
**Pull request by DarkVisenya**

### CODE REVIEW

File: src/modules/Forms/Filters/Filters.js [hunk 1]
```diff
@@ -10,7 +9,7 @@ import Switch from '../Controls/Switch';
 import MultiSelect from '../Controls/MultiSelect';
 import DateRangeInput from '../Controls/DateRangeInput';
 import Control from './Control';
-import translateMock from '../../../utils/translate';
+import translateTerra from '../../../utils/translate';
```
Comment (SebCorbin) [inline src/modules/Forms/Filters/Filters.js]: Why change this name?

File: src/modules/Forms/Filters/Control.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ export class Control extends React.Component {
 
     return (
       <Component
-        translate={translate === defaultTranslate ? undefined : translate}
```
Comment (SebCorbin) [inline src/modules/Forms/Filters/Control.js]: Please check that this works in storybook before removing this

File: src/modules/Forms/Filters/Control.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ export class Control extends React.Component {
...
+        translate={translate}
```
Comment (mabhub) [inline src/modules/Forms/Filters/Control.js]: If you don't transform `translate` anymore, just let it inside spread props:

```jsx
  render () {
    const { component: Component, ...props } = this.props;
    const { values } = this.state;

    return (
      <Component
        {...props}
        values={values}
      />
    );
  }

```

File: src/modules/Forms/Filters/Control.js (hunk 2 above)
Comment (DarkVisenya) [reply src/modules/Forms/Filters/Control.js]: I already test on storybook. It's why I had to mock translate in Select and Filters.

File: src/modules/Forms/Filters/Filters.js (hunk 1 above)
Comment (DarkVisenya) [reply src/modules/Forms/Filters/Filters.js]: An error, sorry

File: src/modules/Forms/Filters/Control.js (hunk 3 above)
Comment (DarkVisenya) [reply src/modules/Forms/Filters/Control.js]: Done

**Pull request merged**